widgetssdk/src/main/java/com/glia/widgets/view/head/ChatHeadView.kt

```diff
@@ -91,9 +94,11 @@ class ChatHeadView @JvmOverloads constructor(
 
     override fun showUnreadMessageCount(unreadMessageCount: Int) {
         post {
-            binding.chatBubbleBadge.apply {
-                text = unreadMessageCount.toString()
-                isVisible = isDisplayUnreadMessageBadge(unreadMessageCount)
+            if (!isCallVisualizerUseCase.invoke()) {
+                binding.chatBubbleBadge.apply {
+                    text = unreadMessageCount.toString()
+                    isVisible = isDisplayUnreadMessageBadge(unreadMessageCount)
+                }
```

Comment:
I would add a small comment as it might not be very intuitive that Call Visualizer does not support chat so it should never display an unread message indicator. Or alternatively, you could make the code more explicit about this behavior e.g:
```suggestion
            if (isCallVisualizerUseCase.invoke()) {
                binding.chatBubbleBadge.isVisible = false
            } else {
                binding.chatBubbleBadge.apply {
                    text = unreadMessageCount.toString()
                    isVisible = isDisplayUnreadMessageBadge(unreadMessageCount)
                }
```

Eather option is fine.